sycl/include/CL/sycl/detail/pi.h

```diff
@@ -301,6 +301,7 @@ typedef enum {
   PI_DEVICE_INFO_IMAGE_SRGB = 0x10027,
   PI_DEVICE_INFO_ATOMIC_64 = 0x10110,
   PI_DEVICE_INFO_ATOMIC_MEMORY_ORDER_CAPABILITIES = 0x10111,
+  PI_DEVICE_INFO_GPU_HW_THREADS_PER_EU = 0x10120,
```

Comment:
why did you leave the gap? why not 0x10112?

Reply:
Hm, I was under impression, that this value is claimed by one of other CL_ definition, but it does look like I was wrong or misinterpreted compilation error.